src/pages/Header/Header.js

```diff
@@ -48,13 +48,20 @@ const Header = () => {
             dataJson.then((data) => {
               // eslint-disable-next-line
               // let zone = window.location.search.match(/zone=([^\^]+)/); // for automating Information Sheet PDFs
-              const { zone } = data;
+              let { zone } = data;
               let match = false;
 
+              let regionId = null;
+
+              if (zone !== '8a' && zone !== '8b') {
```

Comment:
im curious about this exception for just 8a and 8b.  Will the same issue not apply to 7a/b or any others?

Reply:
Yeah I talked to Trevor and the south split zone 8 into 8a and 8b but all other zones get truncated to just the number